sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/administration/DocumentModelAdministrationClientTest

```diff
@@ -28,7 +28,6 @@ import org.junit.jupiter.params.provider.MethodSource;
 
 import java.util.Arrays;
 import java.util.List;
```

Comment:
I can't tell with the diff, do we keep any tests where we do set the custom modelId and pass through the BuildModelOptions?